spec/controllers/reports_controller_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe ReportsController, type: :controller do
+  let!(:user){ create :user }
```

Comment:
no need to exclamation mark as user variable is used in code so it will be created automatically